src/App.css

```diff
@@ -172,6 +155,20 @@ li.panel .button-list {
   max-width: 40%;
 }
 
+aside .section {
```

Comment:
a vertical scroll bar now appear on small screen (width < 1370px). Can we do something about it ?